src/engine/objects/resolvers/visitors.rs

```diff
@@ -1447,7 +1578,20 @@ where
 
     let rel_label = rel_var.src().label()?.to_string() + &rel_var.label().to_title_case() + "Rel";
     let input = if let Some(handlers) = context.event_handlers().before_rel_update(&rel_label) {
-        handlers.iter().try_fold(input, |v, f| f(v))?
+        for f in handlers.iter() {
+            input = f(
+                input,
+                EventFacade::new(
+                    // TODO: where do we get rel_name ?
+                    CrudOperation::UpdateRel(info.name().to_string(), "?".to_string()),
```

Comment:
The RelQueryVar struct has a function to obtain it, so it'd likely be rel_var.label().

Reply:
Fixed.